org.freecad.FreeCAD.yaml

```diff
@@ -1,7 +1,7 @@
 app-id: org.freecad.FreeCAD
 app-id: org.freecad.FreeCAD
 runtime: org.kde.Platform
-runtime-version: '6.7'
+runtime-version: '6.8'
```

Comment:
6.9 is the latest.

Reply:
and you need to use llvm16 in both cases.

Reply:
looks like 24.08 runtime does not have a llvm16 extension, is it included on it? where do I even find the source for the freedesktop sdk anyways? on the flathub account here on github I only see extensions

Reply:
oh. I'm much behind in versions. 18 is the minimum one.